apprentice-react-o-shop/src/components/NavBar/NavBar.js

```diff
@@ -5,7 +5,7 @@ import DropDownItem from './DropDownItem'
 
 function NavBar(){
 
-    const [open, setOpen] = useState(false)
+    const [dropDownOpenBoolean, setdropDownOpenBoolean] = useState(false)
```

Comment:
Lets adjust this naming to be a little more clear like we just talked about in the presentation.